11-model-comparison/11_model_comparison.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p>I would skip the mask cat joke if only because it's a politicized topic and seems like a pointless potential liability. At any rate it slightly irritated me. If the masked cat absolutely must stay at least make the picture smaller and less obtrusive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>